core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -266,7 +266,7 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
             if (!nativeConfig.enableIsolates) {
                 command.add("-H:-SpawnIsolates");
             }
-            if (nativeConfig.enableJni) {
+            if (nativeConfig.enableJni || graalVMVersion.isPresent() && !graalVMVersion.get().contains(" 19.2.1 ")) {
```

Comment:
Please add parentheses here. And I would have ` 19.2.` as the pattern.

Reply:
Does this mean we officially support GraalVM 19.2.0?

Reply:
No. But I don't see the point of being too picky about this while we can just enable JNI for all 19.2.